app/window.go

```diff
@@ -525,4 +526,20 @@ func MinSize(w, h unit.Value) Option {
 	}
 }
 
+// StatusBarColor sets the color of the status bar (which usually contains notification icons
+// and system icons).
+func StatusBarColor(color color.NRGBA) Option {
+	return func(opts *wm.Options) {
+		opts.StatusColor = &color
+	}
+}
+
+// NavigationBarColor sets the color of the navigation bar (which contains the navigation controls,
+// such as "Back", "Home" or "Website Address Bar").
```

Comment:
I think it's better to be specific:

// NavigationBarColor sets the color of the navigation bar on Android, or the address bar in browsers.

Consider shortening to just "NavigationColor".